src/Page.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Page from './Page';
+
+describe('Page에서', () => {
+  const onChangeTitle = jest.fn();
+  const onClickAddTask = jest.fn();
+  const onClickDeleteTask = jest.fn();
+
+  beforeEach(() => jest.clearAllMocks());
+
+  function renderPageWith(
+    taskTitle = '',
+    tasks = '',
```

Comment:
```suggestion
    tasks = [],
```
tasks는 배열이어야 하지 않나요?

Reply:
그렇습니다! 기본값을 사용할 경우가 없어서 몰랐습니다! 바로 수정하겠습니다!😊